docs/configuration.md

```diff
@@ -378,78 +364,84 @@ You can also change the default selection using the [`include`](settings.md#incl
 
 ## Jupyter Notebook discovery
 
-Ruff has built-in support for [Jupyter Notebooks](https://jupyter.org/).
+Ruff has built-in support for linting and formatting [Jupyter Notebooks](https://jupyter.org/).
 
 !!! info
-    Notebooks are linted and formatted by default when using [preview mode](preview.md).
-    You can opt-out of notebook linting and formatting by adding `*.ipynb` to [`extend-exclude`](settings.md#extend-exclude).
+    Jupyter Notebook support is marked as stable from Ruff version `0.6.0` onwards and will
+    be linted and formatted by default.
```

Comment:
The back reference of the second subsentence is unclear. It should refer to Jupyter files but the first sentence only mentions notebook support
```suggestion
    Jupyter Notebook support is marked as stable from Ruff version `0.6.0` onwards and Jupyter Notebooks are linted and formatted by default.
```